.github/workflows/run_chat_analyze.yaml

```
@@ -0,0 +1,46 @@
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Create .env file
+      run: |
+        echo "API_KEY=${{ secrets.GOOGLE_API_KEY }}" >> .env
+
+    - name: Ensure directory exists
+      run: |
+        mkdir -p csv_files charts
+        
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)